nanoFrameworkDeployer/Program.cs

```diff
@@ -249,32 +249,34 @@ private static void RunLogic(CommandlineOptions o)
             }
         }
 
-        private static List<byte[]> CreateBinDeploymentFile(string[] peFiles)
+        internal static List<byte[]> CreateBinDeploymentFile(string[] peFiles)
         {
-            // Keep track of total assembly size
-            long totalSizeOfAssemblies = 0;
-            List<byte[]> assemblies = new List<byte[]>();
-            // now we will deploy all system assemblies
-            foreach (string peItem in peFiles)
+            _message.Verbose("Merging PE assembilies to create single deployment file...");
+            // Keep track of total file binary size
+            long deploymentFileSizeInBytes = 0;
+            List<byte[]> deploymentFileBytes = new List<byte[]>();
+            // now we will add all pe files to create a deployable file
+            foreach (var peFile in peFiles)
             {
-                // append to the deploy blob the assembly
-                using (FileStream fs = File.Open(peItem, FileMode.Open, FileAccess.Read))
+                // append to the pe blob to the deployment bundle
```

Comment:
Your sugestion sounds a bit off... maybe:
```suggestion
                // append the PE file to the deployment bundle
```